packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/lib_auth/blocs/user_account_bloc.dart

```diff
@@ -63,7 +64,14 @@ class UserAccountBloc extends $UserAccountBloc {
   ConnectableStream<bool> _mapToLoggedInState() => Rx.merge([
         _coordinatorBloc.states.isAuthenticated,
         _authService.isAuthenticated().asStream(),
-      ]).publishReplay(maxSize: 1);
+      ]){{#enable_pin_code}}
+      .doOnData((isUserLoggedIn) {
+        if (!isUserLoggedIn) {
+          _coordinatorBloc.events.deleteStoredPin();
+          _coordinatorBloc.events.checkIsPinCreated();
```

Comment:
I use it so that this state:
```
RxBlocBuilder<PinBlocType, bool>(
        state: (bloc) => bloc.states.isPinCreated,
```
in the testapp.dart file gets updated with the `false` value and when the user logs in again the :

The app does not go into this case:

```
if ((loggedIn.data!) &&
                  (isPinCreated.hasData && isPinCreated.data!)) {
                context
                    .read<PinBlocType>()
                    .events
                    .setSessionState(SessionState.startListening);
```